tsconfig.base.json

```diff
@@ -16,7 +16,7 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "downlevelIteration": true,
-    "lib": ["es2015", "es2016", "es2017", "esnext.array", "esnext.asynciterable", "dom"],
+    "lib": ["es2015", "es2016", "es2017", "esnext.string","esnext.array", "esnext.asynciterable", "dom"],
```

Comment:
did you verify that we polyfill these?

Reply:
Not _everything_, but I tested `replaceAll`. What else should I look for?

Reply:
we basically need to make sure that it works in any browser we support (not just local) - so either it's been implemented in all that we support, or safer - that core-js polyfills replaceall (I think that's all that comes in here, right?)

Reply:
Ah, that I did check.

We are good on all modern browsers. 

https://caniuse.com/?search=replaceAll